tests/utils-shared/src/main/java/org/keycloak/testsuite/util/oauth/AbstractOAuthClient.java

```diff
@@ -32,6 +32,7 @@ public abstract class AbstractOAuthClient<T> {
     protected String prompt;
     protected StateParamProvider state;
     protected String nonce;
+    protected String idTokenHint;
```

Comment:
@stianst do you plan to move also the method idTokenHint? https://github.com/keycloak/keycloak/blob/1a5a65330d2e3bb193137f9e1f160af1d04f1437/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/oauth/OAuthClient.java#L532